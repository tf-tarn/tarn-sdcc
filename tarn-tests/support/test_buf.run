./sim/sim -q -h -s 500 $program_file > $output