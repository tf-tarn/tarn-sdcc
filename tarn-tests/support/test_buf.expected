OUTPUT=65
OUTPUT=64
OUTPUT=63
OUTPUT=62
OUTPUT=61
OUTPUT=00
